src/main/scala/com/fulcrumgenomics/commons/reflect/ReflectionUtil.scala

```diff
@@ -396,13 +396,15 @@ object ReflectionUtil {
         val ctor: Constructor[_] = clazz.getDeclaredConstructor(classOf[String])
         ctor.setAccessible(true)
         ctor.newInstance(value)
-      }.getOrElse {
+      }.orElse( Try {
```

Comment:
I'm a little confused here, and think this probably needs tightening up.  E.g. what if in the above block there _is_ a String constructor but the value is invalid?  E.g. `clazz` == Double and values == `"hello"`.